charts/topolvm/values.yaml

```
@@ -157,16 +157,16 @@ lvmd:
   nodeSelector: {}
 
   # lvmd.volumes -- Specify volumes.
-  volumes:
-    - name: lvmd-socket-dir
-      hostPath:
-        path: /run/topolvm
-        type: DirectoryOrCreate
+  volumes: {}
```

```suggestion
  volumes: []
```

The type of value of `volumes` is array, so `[]` is would be appropriate. It seems there is a similar problem in a few places.